srcpkgs/nutty/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'nutty'
+pkgname=nutty
+version=1.1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config vala glib-devel"
+makedepends="gtk+3-devel libgee08-devel libnotify-devel granite-devel
+ sqlite-devel"
+short_desc="Network Utility"
+maintainer="linarcx <linarcx@riseup.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/babluboy/nutty"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=b6c9ef1966d1c60480943313f20cd66ee1b5d23ac8d6578f457fb99f0898d9ba
+
+post_install() {
+	vinstall ${FILESDIR}/com.github.babluboy.nutty.desktop 644 usr/share/applications
```

Comment:
don't bundle this file if it's contained in [the sources](https://github.com/babluboy/nutty/blob/286878fe6d932932ad58da6bb0029f62589a5acc/data/com.github.babluboy.nutty.desktop)